### PR TITLE
Added fourth paragraph

### PR DESCRIPTION
**Pull request by duser1278**
this fourth paragraph added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@ <h1>My First Heading</h1>
 
 <p>My Second paragraph.</p>
 <p>My third paragraph.</p>
+<p>My fourth paragraph.</p>
```
Comment (learnigbyganesh) [inline index.html:10]: what is use of this?

Comment (learnigbyganesh) [inline index.html:10]: what is use of this change?

Comment (learnigbyganesh) [inline index.html:10]: what is use of this?

Comment (duser1278) [reply index.html:10]: this is for demo

Comment (learnigbyganesh) [reply index.html:10]: ok

**Review by learnigbyganesh — APPROVED** [review]
ok all good

**Pull request merged**